src/Microsoft.Azure.NotificationHubs/XiaomiCredential.cs

```diff
@@ -0,0 +1,148 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for 
+// license information.
+//------------------------------------------------------------
+using System;
+using System.Runtime.Serialization;
+using Microsoft.Azure.NotificationHubs.Messaging;
+
+namespace Microsoft.Azure.NotificationHubs
+{
+    /// <summary>
+    /// Represents a Xiaomi credential.
+    /// </summary>
+    [DataContract(Name = ManagementStrings.XiaomiCredential, Namespace = ManagementStrings.Namespace)]
+    public class XiaomiCredential : PnsCredential
```

Comment:
If implementing equality checks, you should conform to the standard way of doing [Equality in C#](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/how-to-define-value-equality-for-a-type)